volumes/short_term_counting_program/sql/create-table-mto_length_bin_classification.sql

```diff
@@ -0,0 +1,30 @@
+--DROP TABLE IF EXISTS traffic.mto_length_bin_classification;
+
+CREATE TABLE traffic.mto_length_bin_classification
```

Comment:
where do the numbers come from? also can you fluff a liiitle + keep the decimal places consistent in the numrange

Reply:
This image... don't have a link to the actual source document but I'll stick it in the readme! 💡 
![image](https://github.com/user-attachments/assets/6833eca0-ebfe-47ab-8dd9-77cc07f47e1e)